packages/app/config/environment.js

```diff
@@ -59,11 +58,12 @@ module.exports = function(environment) {
     ENV.APP.LOG_VIEW_LOOKUPS = false;
 
     ENV.APP.rootElement = '#ember-testing';
+    ENV.APP.autoboot = false;
   }
 
   if (environment === 'production') {
     ENV['ember-cli-mirage'] = {
-      enabled: false //set to true for gh-pages
+      enabled: true //set to true for gh-pages
```

Comment:
Is this change intentional?

Reply:
see comment, wondering why it was ever set to false.

Reply:
For the `gh-pages` demo we need to enable the mock for `prod` builds manually. However, as a user of the Navi app, when you clone the repo and configure your web services, you would want to disable mirage for prod builds.